real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/LoggingNode.java

```diff
@@ -152,7 +152,9 @@ private void setupSubscriberList() {
             new Subscriber(filter.getMsgPath(), new MessageListener() {
                 @Override
                 public void actionPerformed(String topicName, Message m) {
-                    messageQueue.add(m);
+                    if (keepLogging) {
```

Comment:
Making it a bounded queue leaves us open to possibly dropping packets. This way means that we probably won't drop any packets.